### PR TITLE
apt.conf.d

### PR DESCRIPTION
**Pull request by ze42**
apt_conf management

* Rebased #17
* Added missing default dict in `map.jinja` for `apt_conf_d`
* Handling `remove_apt_conf` like other `remove_*` options

### CODE REVIEW

**Comment by javierbertoli** [comment]
@ze42, can you rebase again? The branch has conflicts with master.

**Comment by ze42** [comment]
@javierbertoli ummm. Sorry, I don't see any conflict with master...

Master seems to be at 35cf2e0, which is 5 commit behind this PR.

Plus, it states "This branch has no conflicts with the base branch"...

**Comment by javierbertoli** [comment]
> @javierbertoli ummm. Sorry, I don't see any conflict with master...
>Master seems to be at 35cf2e0, which is 5 commit behind this PR.
> Plus, it states "This branch has no conflicts with the base branch"...

@ze42 you're right, now GH shows me no conflict on this branch. My bad I guess, I didn't check it on the CLI, just trusted what the webUI was showing me. Sorry. :yum:

File: apt/map.jinja [hunk 1]
```diff
@@ -14,22 +14,25 @@
...
         'default_url': 'http://deb.debian.org/debian/',
         'default_keyring_package': 'debian-archive-keyring',
         'repositories': {
           'sane_default': {
-             'distro': distribution,
-             'url': 'http://deb.debian.org/debian/',
-             'arch': arch,
-             'comps': ['main'],
+            'distro': distribution,
+            'url': 'http://httpredir.debian.org/debian/',
```
[the remark this answers is not in the document]
Comment (ze42) [reply apt/map.jinja]: rebased, removing those unrelated changes from original commits.

**Comment by aboe76** [comment]
@ze42 thanks for this, @javierbertoli thanks for reviewing.

**Pull request merged**